.mailmap

```diff
@@ -1,15 +1,12 @@
 # For the purpose of this file, see:
 # https://timvink.github.io/mkdocs-git-authors-plugin/mailmap.html
-Christos Varelas <christos.varelas@citynetwork.eu>
+Christos Varelas <christos.varelas@cleura.com> <christos.varelas@citynetwork.eu>
 Christos Varelas <christos.varelas@cleura.com>
-Dmitriy Rabotyagov <dmitriy.rabotyagov@citynetwork.eu>
-Dmitriy Rabotyagov <dmitriy.rabotyagov@cleura.com>
+Dmitriy Rabotyagov <dmitriy.rabotyagov@cleura.com> <dmitriy.rabotyagov@citynetwork.eu>
 Eliott Trouillet <53047591+packettoobig@users.noreply.github.com>
-Florian Haas <florian@citynetwork.eu>
-Florian Haas <florian@cleura.com>
-Foad Lind <foad.lind@citynetwork.eu>
-Foad Lind <foad.lind@cleura.com>
-Mateusz Guziak <mateusz.guziak@cleura.com>
-Mateusz Guziak <skipper126@interia.pl>
-Namrata Sitlani <namrata.sitlani@citynetwork.eu>
-Namrata Sitlani <namrata.sitlani@cleura.com>
+Florian Haas <florian@cleura.com> <florian@citynetwork.eu>
+Florian Haas <florian@cleura.com> <florian.haas@cleura.com>
```

Comment:
Can you have a single line with one full name and three email addresses?

Reply:
Nope, that's not how `.mailmap` works. See [the documentation](https://git-scm.com/docs/gitmailmap) (if you can parse the famously impenetrable git manpage language; see also <https://git-man-page-generator.lokaltog.net> for comic relief).